starter-code/apple-pie-starter-code/main.css

```diff
@@ -0,0 +1,81 @@
+/* ---- typography ---- */
```

Comment:
Consider starting your css reseting some styles. That way you reduce browser inconsistencies. We will see more advanced stuff. For now, you can do:
```
/* ---- reset ---- */
h1, body, html, main {
  margin: 0;
}
```